components/list/list.js

```diff
@@ -69,7 +80,7 @@ class List extends LitElement {
 
 	_getItems() {
 		return this.shadowRoot.querySelector('slot').assignedNodes().filter((node) => {
```

Comment:
We don't need to add it right away, but something to be mindful/careful of is - slotted elements of the children's slots that are being distributed to this slot (nested slots).  To resolve that scenario, the flatten option can be passed.  `slot.assignedNodes({flatten: true})`.  Not something that is needed right away, but definitely something that could come up depending on how the consumer is composing the child items.